### PR TITLE
chore(workflow): add action for duplicate issue

### PR DESCRIPTION
**Pull request by xzf0587**

### CODE REVIEW

File: .github/workflows/duplicate-app-studio-service-issue.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: duplicate issue check for app-studio-service 
+on:
+  issues:
+    types: [opened]
```
Comment (wenytang-ms) [inline .github/workflows/duplicate-app-studio-service-issue.yml:4]: I think update if needed?
if user update the title or update the description?

Comment (xzf0587) [reply .github/workflows/duplicate-app-studio-service-issue.yml]: Updated. Thanks.

File: .github/workflows/duplicate-app-studio-service-issue.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: "OfficeDev/teamsfx-issue-triage-bot"
+          ref: main
```
Comment (wenytang-ms) [inline .github/workflows/duplicate-app-studio-service-issue.yml:14]: we always make code change on dev, and looking forward to revisions and updates.
so this ref may be dev? so checkout the latest change in actions files?

Comment (xzf0587) [reply .github/workflows/duplicate-app-studio-service-issue.yml:14]: The OfficeDev/teamsfx-issue-triage-bot repo is a simple project.
It is better to use the stable code in main branch.

**Review by wenytang-ms — APPROVED** [review]

**Pull request merged**